pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -225,7 +225,13 @@ func (s *Service) getOrUpdateAllowedCIDRS(openStackCluster *infrav1.OpenStackClu
 		allowedCIDRs = append(allowedCIDRs, openStackCluster.Spec.APIServerLoadBalancer.AllowedCIDRs...)
 
 		if openStackCluster.Spec.Bastion.Enabled {
-			allowedCIDRs = append(allowedCIDRs, openStackCluster.Status.Bastion.FloatingIP, openStackCluster.Status.Bastion.IP)
+			if openStackCluster.Status.Bastion.FloatingIP != "" {
```

Comment:
will it still nil point? I think in issue you mentioned `openStackCluster.Status.Bastion` is empty at this point of time?

Reply:
Yes you're right. Only the ready field is set in  the initial reconciliation phase. Will fix this